scripts/fetchai_netutils/fetch/testing/testcase.py

```diff
@@ -0,0 +1,477 @@
+import os
+import json
+import time
+import datetime
+import glob
+import shutil
+import subprocess
+
+from fetch.cluster.instance import ConstellationInstance, DmlfEtchInstance
+from fetch.cluster.utils import output, verify_file
+
+from fetchai.ledger.crypto import Entity
+
+
+class TestCase(object):
+    """
+    """
+
+    def append_node(self, index, load_directory=None):
+        pass
+
+    def connect_nodes(self, node_connections):
+        pass
+
+    def start_node(self, index):
+        pass
+
+    def restart_node(self, index):
+        pass
+
+    def setup_input_files(self):
+        pass
+
+    def run(self):
+        pass
+
+    def stop(self):
+        pass
+
+    def print_time_elapsed(self):
+        pass
```

Comment:
Could some of this common logic go in the parent class? printing time, dumping logs, setting up a workspace for example

Reply:
yes, definitely, that is the plan